Game/Assets/Coliseo/Scripts/Actor.cs

```diff
@@ -0,0 +1,140 @@
+﻿using UnityEngine;
+using System.Collections;
+
+namespace Coliseo
+{
+    public abstract class Actor : MonoBehaviour
+    {
+        protected const int MAX_HEALTH = 100;
+        protected const int attackStrength = 30;
+        public const float moveSpeed = 6f;
+        public const float jumpStrength = 50.0f;
+
+        protected int _health;
+
+        protected TextMesh healthDisplay;
+
+        protected Animator anim;
+        protected SaberController saberCont;
+        protected Rigidbody rb;
+        protected GameObject saber;
+
+        public int health
+        {
+            get { return _health; }
+            set
+            {
+                _health = value;
+
+                if (isDead)
+                {
+                    this.die();
+                }
+
+                healthDisplay.text = "" + health;
+            }
+        }
+
+        public bool isDead
+        {
+            get { return health <= 0; }
+        }
+
+        bool damaged {
+            get { return health != MAX_HEALTH; }
```

Comment:
Can we ever have more than `MAX_HEALTH`? Powerups? This may seem like splitting at straws, but a bug like that would be difficult to hunt down I think.